src/main/java/Duke.java

```diff
@@ -1,10 +1,102 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static void greetUser() {
+         String divider = "______________________________";
+         String greeting = divider + System.lineSeparator() + "Hello! I'm Jarvis!"
+                 + System.lineSeparator() + "What can I do for you?"
+                 + System.lineSeparator() + divider + System.lineSeparator();
+         System.out.println(greeting);
+    }
+
+    public static void echoUserInput(String curr) {
+        String divider = "______________________________";
+        String echo = divider +System.lineSeparator() + "added: " + curr + System.lineSeparator()
+                + divider + System.lineSeparator();
+        System.out.println(echo);
+    }
+
+    public static void sayByeToUser() {
+        String divider = "______________________________";
+        String bye = divider + System.lineSeparator() + "Bye. Hope to see you again soon!"
+                + System.lineSeparator() + divider + System.lineSeparator();
+        System.out.println(bye);
+    }
+
+    public static void addToList(Task[] tasks, String curr, int taskSum) {
+        tasks[taskSum] = new Task(curr);
+    }
+
+    public static void printList(Task[] tasks, int taskSum) {
+        String divider = "______________________________";
+        System.out.println(divider + System.lineSeparator()
+                + "Here are the tasks in your list:");
+        for (int i = 0; i < taskSum; i++) {
+            System.out.println((i+1) + ". " + "[" + tasks[i].getStatusIcon() + "] " + tasks[i].getName());
+        }
+        System.out.println(divider);
+    }
+
+    public static void markAsDone(Task[]tasks, int taskNum, int taskSum) {
+        String divider = "______________________________";
+        int taskIndex = taskNum - 1;
+        if (taskNum > taskSum) {
```

Comment:
This can be refactored since it has been repeated a couple times in the code. It makes maintaining and editing changes much easier since you just need to edit it once.
https://nus-cs2113-ay2223s2.github.io/website/se-book-adapted/chapters/codeQuality.html#minimise-code-duplication